src/core/server/base.js

```diff
@@ -6,10 +6,19 @@ import React from 'react';
 import { renderToString } from 'react-dom/server';
 import { RouterContext, match } from 'react-router';
 
+import config from 'config';
+import devServer from './dev';
+
+const ENV = config.get('env');
+
 export default function (routes) {
```

Comment:
Shouldn't this fail eslint now? I guess it just isn't checking for the space now. So the config should be `"space-before-function-paren": "never"` maybe?